devices/tplink_re200-v4.yml

```diff
@@ -0,0 +1,23 @@
+vendor: TP-Link
+model: RE200
+variant: v4
+bootloader: unknown
+cpu: MediaTek MT7628AN
+cpu_cores: 1
+cpu_mhz: 580
+memory_mb: 64
+storage:
+  - type: unknown
+    size_mb: 8
+device_type: WiFi Router
+ethernet_100m_ports: 1
+jtag: false
+serial: 57600 / 8N1
+outdoor: false
+supported_current_rel: none
+supported_since_rel: snapshot
+target: ramips
```

Comment:
Here, we should discuss whether we really want to put all devices into the same folder. This becomes particularly interesting when you have situations like ar71xx/ath79 transition, where the device yml file normally won't change. This would require either duplicating the file into two target subdirectories, or naming two targets in a single file.

This looks rather open to me at the moment.

Reply:
We could use an array here as well. After all it's the same device, how we structure the target/subtargets doesn't matter. I'm against sub folders for targets which are then moved around after e.g. ar71xx deprecation.

Reply:
I'm not sure yet whether not having target subfolders is actually practical here.

However, by not having subfolders and using the target variable this could actually provide an additional benefit by providing a device name to target mapping for the user (or whatever user script is parsing this folder).

Reply:
Maybe it's better to use the full target like `ath79/generic` instead of just the *target*, I think some devices are available in multiple subtargets(?).

Reply:
I also think "full" information (target and subtarget) should be provided here.

If devices are in multiple subtargets, we could either only show the target, or, (preferably) just list both, e.g. bcm63xx/generic and bcm63xx/smp like we would list ath79/generic and ar71xx/generic as well.

Reply:
It's like that now. Full target, an array